packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -86,6 +89,18 @@ def get_spans_dataframe(
 
             _ = pd  # Prevent unused symbol error
 
+            if project_identifier and project_name:
+                raise ValueError("Provide only one of 'project_identifier' or 'project_name'.")
+            elif project_identifier and not project_name:
+                project_response = self._client.get(
+                    url=f"v1/projects/{project_identifier}",
+                    headers={"accept": "application/json"},
```

Comment:
not clear why we need these headers. Don't you need the authentication headers to propagate here. This is additive?

Reply:
yeah these headers are additive and we need this header to not get pyarrow back